test/TestPrisonersDilemma.js

```diff
@@ -7,7 +7,10 @@ var EMPTY_ADDRESS = "0x0000000000000000000000000000000000000000";
 contract('PrisonersDilemma', async (accounts) => {
 
     beforeEach(async() => {
-        instance = await PrisonersDilemma.new(accounts[0], accounts[1]);
+        instance = await PrisonersDilemma.new(
```

Comment:
Move this out since this doesn't apply to each instance of a test.

Reply:
In addition to this reduce the existing tests so they are more succinct.

Reply:
Nvm I am just going to overwrite the instance. I just need to ensure that if this were tested on a live network that I'd be deploying the same contract multiple times and spend a lot of money.